### DISCUSSION TITLE
你曾经在外星球上居住过一段时间吗？

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [share121/inter-knot#1649] [公告]
![x](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.9qj99lczolLnVBgAascggwAAAA%26pid%3DApi&f=1&ipt=8dcbb17973849156eb51606e7146fe68d0933ac3a690f1ec8d9045d34d4486ec&ipo=images)

你曾经在外星球上居住过一段时间吗？

**Comment by gaojiaobei** [comment]
是的，我曾经在一颗名为雅利洛-Ⅵ的星球居住甚至工作过一段时间
那里终年冰天雪地，但当地原住民却与我们有着同样甚至更甚的热情

**Comment by kallen1314** [comment]
我曾经跟热心的邻家姐姐一起去月球旅行过，全程免费呢，还交到了好多新朋友。欢迎大家有机会去新亚特拉玩！

**Comment by icenow188** [comment]
是的，并且现在还在，不是我不想回去，我已经快忘了我待在这里多久了，还没找到我哥（如果有人看到一个扎辫子的黄毛请联系我）

**Comment by cpu644** [comment]
⑩得。，我噌劲再特兰忒与大名顶顶的哈利-谢尔吨共事。

**Comment by ikungei** [comment]
I once went to a country located on the continent of Tiwate, called Liyue. It has a vast land, rich resources, and cultural landscapes similar to China on Earth. The people there are warm and friendly, and the scenery is beautiful and moving, like a replica of China. It is said that there is also a god named Morax there, but I was unable to see his true face. It is probably the only regret of my trip. If given the opportunity, we would be willing to visit that country again
